Source/Models/FORMField.m

```diff
@@ -224,9 +231,9 @@ - (FORMValidationResultType)validate
 {
     id validator;
     Class validatorClass;
-
+    
```

Comment:
Trailing spaces are the worst, please consider using Marvin, I know @attila-at-hyper, @zenangst and me use it (it's available on Alcatraz).

https://github.com/zenangst/MarvinXcode/issues/13